backpack/extensions/secondorder/hbp/__init__.py

```diff
@@ -115,6 +128,21 @@ def __init__(self, mc_samples=1):
     def get_num_mc_samples(self):
         return self._mc_samples
 
+    def accumulate_backpropagated_quantities(
+        self, existing: Tensor, other: Tensor
+    ) -> Tensor:
+        """Accumulate backpropagated tensors by summation.
+
+
+        Args:
+            existing: first backpropagated tensor.
+            other: second backpropagated tensor.
+
+        Returns:
+            Summation of the two tensors.
+        """
+        return existing + other
+
```

Comment:
Can you move this to the `KFAC` extension? We need tests for the other Kronecker curvatures (KFRA, KFLR) that inherit from `HBP` before officially supporting them in ResNets. But these are not of high priority at the moment.

Reply:
You are right, the KFAC extension is `KFAC(HBP)`

Reply:
I believe it is already in the KFAC extension--do you mean to add something to HBP?

Reply:
Yes, sorry for the confusion. Closing.